app/lib/cms/s3.rb

```diff
@@ -32,11 +32,15 @@ def credentials
       CMS::AwsCredentials.instance.credentials
     end
 
+    def hostname
```

Comment:
Would it make sense to just add `.presence` on line :45 ? i.e.
````
    %i[access_key_id bucket hostname region role_arn role_session_name secret_access_key web_identity_token_file].each do |param|
      define_method(param) do
        config[param].presence if enabled?
      end
```

Reply:
That'd return `nil` for all configurations that are not present.
I don't know if it'd make sense for all available configurations, to be honest.
In the past, only `protocol` and `hostname` used `.presence` so, In this PR, I'm just reverting that change. As for the other configurations, we'd need to check one by one if there could be potential bugs on passing nils instead of empty strings.

I'd leave those behaving the same it was in the past for now and just fix the one that's causing problems.

Reply:
Yeah, I thought it would make sense to have the same behavior for all.
But agreed that we can review this later.

Reply:
Updated in 28bea41.